decentraland/components/presentational/LandForSaleInfo.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { StyleSheet, View, Text, Alert } from "react-native";
+import PropTypes from "prop-types";
+import {
+  responsiveWidth,
+  responsiveFontSize,
+} from "react-native-responsive-dimensions";
+import Colors from "@constants/Colors";
+import { Button, Icon } from "native-base";
+import { formatNumber } from "../../helpers";
+
+const onPriceInfo = () => {
+  const title = "";
+  const message =
+    "MANA is Decentraland’s fungible, ERC20 cryptocurrency token. MANA is burned or spent in exchange for LAND parcels. ";
+  Alert.alert(title, message, [{ text: "Okay" }]);
+};
+
+export default function LandForSale({ landForSale }) {
+  const { asset, priceMana } = landForSale;
+  let { name } = asset;
+  const price = formatNumber(priceMana);
+  if (!name) name = "Land without name";
+
+  // Note: Conversion to USD will be implemented on v0.2.0
+  // <Text>{priceMana} MANA (~${landForSale.priceUSD})</Text>
+  return (
+    <View style={styles.landInfoContainer}>
+      <View style={styles.landNameContainer}>
+        <Text style={styles.landName}>{name}</Text>
+      </View>
+      <View style={styles.landPriceContainer}>
+        <View>
+          <Button small transparent onPress={onPriceInfo}>
+            <Icon
+              name="information-circle-outline"
+              style={styles.landPriceIcon}
+            />
+          </Button>
+        </View>
+        <View>
+          <Text style={styles.landPrice}>{price}</Text>
+        </View>
+      </View>
+    </View>
+  );
+}
+
+LandForSale.propTypes = {
+  landForSale: PropTypes.object.isRequired,
+};
+
+const styles = StyleSheet.create({
```

Comment:
I think we may want to ensure top-alignment between the text for the price number and the land/estate name here

Reply:
I've faced the same problem related here: https://github.com/facebook/react-native/issues/8953 and because of that, I've added `paddingTop` just for the estate name.

Reply:
Ah ok - I'll look into a solution that could work well for us. In the meantime, hardcoding padding on top for each chunk of text that roughly top-aligns them can work.

Reply:
Let's put a little more space on top of the price before merging

![TempAlignment](https://user-images.githubusercontent.com/2731712/55420800-7ec95300-553d-11e9-9ec3-bc8c777ba1c6.png)

Reply:
Done.

Reply:
Thanks! Resolving